install_ads.bat

```diff
@@ -0,0 +1,241 @@
+rem e.g.
+rem        install_ads.bat 5.5
+
+echo:
+echo *******************************
+echo * Welcome to ADS installation *
+echo *******************************
+
+rem This option is needed for expanding !git_ref!, which is set (*and expanded*!) inside the 'if' statement below.
+rem See also https://stackoverflow.com/q/9102422 for a further description of this behavior.
+setLocal EnableDelayedExpansion
+
+set TMP_DIR=%temp%\tmp-%RANDOM%%RANDOM%
+mkdir %TMP_DIR%
+
+rem Try to ensure that Git is available on the PATH prior to invoking `git clone` to avoid 'command not found' errors
+rem   - See also: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3912
+rem NB: This *should* be handled by the git installer, and we even have reports of people running git --version
+rem successfully and still getting an error. However, there are perhaps situations where someone has installed git but
+rem hasn't refreshed their terminal. Manually modifying the PATH is a bit of a hacky workaround, especially if Git has
+rem been installed somewhere else, but if this mitigates a user post on the forum, this will save us some dev time.
+PATH=%PATH%;C:\Program Files\Git
+git --version >nul 2>&1 || (
+    echo ### git not found. Make sure that git is installed ^(and a fresh Command Prompt window has been opened^) before running the ADS installer.
+    goto error
+)
+
+rem Default value: 'master', however this value is updated on stable release branches.
+set git_ref=master
+
+rem Check to see if the PWD contains the project source files (using `__init__.py` as a proxy for the entire source dir)
+rem If it exists, then we can reliably access source files (e.g. `requirements-freeze.txt`) from the PWD.
+if exist AxonDeepSeg\__init__.py (
+  set ADS_SOURCE=%cd%
+rem If __init__.py isn't present, then the installation script is being run by itself (i.e. without source files).
+rem So, we need to clone ADS to a TMPDIR to access the source files, and update ADS_SOURCE accordingly.
+) else (
+  set ADS_SOURCE=%TMP_DIR%\axondeepseg
+  echo:
+  echo ### Source files not present. Downloading source files ^(@ !git_ref!^) to !ADS_SOURCE!...
+  git clone -b !git_ref! --single-branch --depth 1 https://github.com/axondeepseg/axondeepseg.git !ADS_SOURCE!
+  rem Since we're git cloning into a TMPDIR, this can never be an "in-place" installation, so we force "package" instead.
+  set ADS_INSTALL_TYPE=package
+)
+
+rem Get installation type if not already specified
+if [%ADS_INSTALL_TYPE%]==[] (
+  rem The file 'requirements-freeze.txt` only exists for stable releases
+  if exist %ADS_SOURCE%\requirements-freeze.txt (
+    set ADS_INSTALL_TYPE=package
+  rem If it doesn't exist, then we can assume that a dev is performing an in-place installation from master
+  ) else (
+    set ADS_INSTALL_TYPE=in-place
+  )
+)
+
+rem Fetch the version of ADS from the source file
+for /F %%g IN (%ADS_SOURCE%\AxonDeepSeg\__init__.py) do (set ADS_VERSION=%%g:~15,20%)

```

Comment:
TODO for self: Fix the version parsing as it doesn't work.

Reply:
@joshuacwnewton, would this be a quick fix to do?

Reply:
Sorry for the delay on this! Working on it now. :)

Reply:
Should be fixed in https://github.com/axondeepseg/axondeepseg/pull/800/commits/5370daaab7de2b320d3dabf61685cd67d4da20f6. :)